lib/cylc/gui/app_gcylc.py

```diff
@@ -175,7 +175,7 @@ def reset(self, suite, auth=None):
             else:
                 self.host = auth
                 self.port = None
-        self.logdir = SuiteLog.get_inst(suite).get_dir()
+        self.logdir = SuiteLog(suite, renew=True).get_dir()
```

Comment:
Just discovered that you can use `SuiteLog.get_dir_for_suite(suite)` (a static method) instead.

Reply:
Good spotting - sensible of us to put that method in!